commands/device/get-file.ts

```diff
@@ -1,15 +1,25 @@
 export class GetFileCommand implements ICommand {
 	constructor(private $devicesService: Mobile.IDevicesService,
 		private $stringParameter: ICommandParameter,
+		private $project: Project.IProjectBase,
+		private $errors: IErrors,
 		private $options: ICommonOptions) { }
 
-	allowedParameters: ICommandParameter[] = [this.$stringParameter];
+	public allowedParameters: ICommandParameter[] = [this.$stringParameter, this.$stringParameter];
 
 	public async execute(args: string[]): Promise<void> {
 		await this.$devicesService.initialize({ deviceId: this.$options.device, skipInferPlatform: true });
+		let appIdentifier = args[1];
```

Comment:
as we modify the command, shouldn't we delete the `$options.app` that has been used before (not in the command itself, but in the service behind it. If I understand correctly, we'll support getting of file only from application sandbox, so the `$options.app` is a mandatory argument now. So... let's remove `$options.app`